README.md

```diff
@@ -0,0 +1,87 @@
+# vo-models
```

Comment:
no badge for your 100% code coverage? :P

speaking of, i saw you were using pytest-cov -- but the coverage report isn't being used anywhere, right? it's just output in the workflow run?

Reply:
Yeah, just output for now. Was thinking about using the coverage report somehow, but decided against it for now. Maybe I'll add that little badge at some point too.